app/routing/routing_engine.py

```diff
@@ -1,10 +1,40 @@
+class RoutingException(Exception):
+    pass
+
+
 class RoutingEngine(object):
     ''' The routing engine will apply any routing rules dependant on where the user is in the schema
     and what answers they have provided.
     '''
     def __init__(self, schema_model):
         self._schema = schema_model
 
-    def get_next(self, current_location):
-        # however we're yet to implement routing so always return the current location
-        return current_location
+    def get_next_location(self, current_location):
```

Comment:
Technically no, but the system is flexible enough to cope with us changing it to question or section for example. That's why we haven't called it get_next_block